_posts/config/2017-05-08-cloud-services.md

```diff
@@ -4,7 +4,7 @@ title:  "Cloud Services"
 categories: [config]
 ---
 
-The Cloud Services section of the Kura Administrative Web UI allows to create and manage cloud connections.
+The Cloud Services section of the Kura Administrative Web UI allows us to create and manage cloud connections.
```

Comment:
Is "us" the proper term to use here?

Reply:
A verb like "allow" cannot follow an infinitive verb (in this case it's "to create") because we have to allow a person or a thing, as far as I know.

Reply:
You are definitely right, the original sentence is wrong. But maybe it's better to reword it

Reply:
I suggest to replace "us" with "user" 😄